INSTALLATION.rst

```diff
@@ -66,14 +66,19 @@ Recommended Method
 
     .. attention::
 
-       Note, this does not work on Windows. On that platform, you should be able to do::
+       Note, this does not work on Windows x86, as conda-forge_ no longer
```

Comment:
Perhaps clarify that it *will* work for Windows x86_64, it's just the 32-bit flavor that's deprecated?

Reply:
Good point. Will clarify.